sentry_sdk/integrations/wsgi.py

```diff
@@ -62,7 +62,14 @@ def get_request_url(environ, use_x_forwarded_for=False):
     return "%s://%s/%s" % (
         environ.get("wsgi.url_scheme"),
         get_host(environ, use_x_forwarded_for),
-        wsgi_decoding_dance(environ.get("PATH_INFO") or "").lstrip("/"),
+        wsgi_decoding_dance(
+            "/".join(
+                [
+                    environ.get("SCRIPT_NAME", "").rstrip("/"),
+                    environ.get("PATH_INFO", "").lstrip("/"),
+                ]
+            )
+        ).lstrip("/"),
```

Comment:
The following might be more readable. For readability, I would also suggest computing the path before the return statement and storing it in a variable (named `path` or something similar); then, the call could be replaced with `wsgi_decoding_dance(path).lstrip("/")`.
```suggestion
        wsgi_decoding_dance(
            "%s/%s" % (
                environ.get("SCRIPT_NAME", "").rstrip("/"),
                environ.get("PATH_INFO", "").lstrip("/"),
            )
        ).lstrip("/"),
```